dashboard/server/api/index.js

```diff
@@ -1,7 +1,33 @@
-module.exports = {
-    testSocketConnection: (client) => {
-        client.on('test-conn', (cb) => {
-            cb('hello world');
-        });
+module.exports = (socket) => ({
+    handleNewRun: (data, callback) => {
+        socket.to('frontend').emit('process:start', data);
```

Comment:
Maybe we pass `socket.to('frontend')` instead of `socket` in the exported function (if we're only emitting to "frontend"? Won't have to write the same thing multiple times.

Reply:
Yes, that is amazing! Will do this!